cypress/support/api_commands.js

```diff
@@ -177,6 +177,44 @@ Cypress.Commands.add('apiAddUserToTeam', (teamId, userId) => {
     });
 });
 
+/**
+ * Join teammates directly via API
+ * This API assume that the user is logged in and has cookie to access
+ * @param {String} name - The name arg previously used during team-create (unique but not GUID)
+ * @param {Integer} size - The amount of users to join
+ * All parameter required
+ */
+Cypress.Commands.add('apiBatchJoinTeam', (name, size = 60) => {
+    return cy.getCookie('MMUSERID').then((cookie) => {
+        const admin = cookie.value;
+        cy.request({method: 'GET', url: `/api/v4/teams/name/${name}`}).then((teamResp) => {
+            const key = teamResp.body.id;
+            cy.request({method: 'GET', url: `/api/v4/users?per_page=${size}`}).then((userResp) => {
+                // Shape the users list into batch request body, skip the team admin.
+                const ulist = userResp.body.filter((u) => u.id !== admin).map((user) => {
+                    return {team_id: key, user_id: user.id, roles: 'system_user', delete_at: 0, scheme_user: true, scheme_admin: true, explicit_roles: 'system_user'};
+                });
+
+                // The API errors when the list is more than 20 users, so do chunks.
+                for (let i = ulist.length; i > 20; i -= 20) {
+                    const twenty = [];
```

Comment:
nit: Can we name this as `userIds` instead?

Reply:
It is a little different with the refactor, but did rename the lists to userIds. The 'twenty' is still the name for a chunk because I got a warning about name collision when I tried to make it userIds.